libshaderc/include/shaderc.h

```diff
@@ -146,11 +157,16 @@ void shaderc_module_release(shaderc_spv_module_t module);
 // Returns true if the result in module was a successful compilation.
 bool shaderc_module_get_success(const shaderc_spv_module_t module);
 
-// Returns the number of bytes in a SPIR-V module.
+// Returns the number of bytes in a SPIR-V module result string. When the module
+// is compiled with disassembly mode or preprocessing only mode, the result
+// string is a char string. Otherwise, the result string is binary string.
 size_t shaderc_module_get_length(const shaderc_spv_module_t module);
 
-// Returns a pointer to the start of the SPIR-V bytes.
-// This is guaranteed to be castable to a uint32_t*.
+// Returns a pointer to the start of the SPIR-V bytes, either SPIR-V binary or
+// char string. When the source string is compiled into SPIR-V binary, this is
+// guaranteed to be castable to a uint32_t*. If the source string is compiled in
+// disassembly mode or preprocessing only mode, the pointer points to the result
+// char string, and it will not castable to a uint32_t*.
```

Comment:
Just leave off the last part of the last sentence: ", and it will not be castable to a uint32_t*".  Becuase it might be doable, but you have no guarantee.   Better to just not say anything.